src/main/webapp/assets/js/view/application-add-wizard.js

```diff
@@ -303,6 +303,7 @@ define([
         },
         previewStep:function () {
             // no need for validation if going to composer
+            this.currentView.validate();
```

Comment:
The comment right above is valid, at that point, we shouldn't validate the model but send exactly what the user specified. So rather than calling validate here, I would add this above line `317`:
```js
this.model.spec.set("config", this.currentStep.getConfigMap());
```